modules/reklamstoreBidAdapter.js

```diff
@@ -0,0 +1,141 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'reklamstore';
+const ENDPOINT_URL = '//ads.rekmob.com/m/prebid';
+const CURRENCY = 'USD';
+const TIME_TO_LIVE = 360;
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['reklamstore'],
+  /**
+  * Determines whether or not the given bid request is valid.
+  *
+  * @param {BidRequest} bid The bid params to validate.
+  * @return boolean True if this is a valid bid, and false otherwise.
+  */
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.regionId);
+  },
+  /**
+  * Make a server request from the list of BidRequests.
+  *
+  * @param {validBidRequests[]} - an array of bids
+  * @return ServerRequest Info describing the request to the server.
+  */
+  buildRequests: function (validBidRequests) {
+    let requests = [];
+    let loc = utils.getTopWindowLocation();
```

Comment:
Use http://prebid.org/dev-docs/bidder-adaptor.html#referrers 
`utils.getTopWindowLocation();`  is a deprecated function and will be removed soon